{"cveId": "CVE-2020-10185", "cwe": ["CWE-294", "CWE-294"], "cvss": [{"baseScore": "8.6", "version": "3.1", "Attack Vector": "None", "Attack Complexity": "Low", "Privileges Required": "None", "User Interaction": "None", "Scope": "Unchaged", "Confidentiality Impact": "Low", "Integrity Impact": "Low", "Availability Impact": "High"}], "references": ["https://www.yubico.com/support/security-advisories/ysa-2020-01/", "https://github.com/Yubico/yubikey-val/releases/tag/yubikey-val-2.40", "https://lists.debian.org/debian-lts-announce/2020/03/msg00014.html"], "description": ["The sync endpoint in YubiKey Validation Server before 2.40 allows remote attackers to replay an OTP. NOTE: this issue is potentially relevant to persons outside Yubico who operate a self-hosted OTP validation service with a non-default configuration such as an open sync pool; the issue does NOT affect YubiCloud."], "published": "2020-03-05T23:15:13.353", "state": "PUBLIC", "vendorName": ["n/a"], "productName": ["n/a"], "github": {"advisories": [], "commits": [], "pocAdvisorie": null, "repo": null, "info": {}}, "pocList": []}